lib/UseStrictPlugin.js

```diff
@@ -2,19 +2,15 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var ConstDependency = require("./dependencies/ConstDependency");
+const ConstDependency = require("./dependencies/ConstDependency");
+const NullFactory = require("./NullFactory");
 
-var NullFactory = require("./NullFactory");
-
-function UseStrictPlugin() {}
-module.exports = UseStrictPlugin;
-
-UseStrictPlugin.prototype.apply = function(compiler) {
-	compiler.plugin("compilation", function(compilation, params) {
-		params.normalModuleFactory.plugin("parser", function(parser) {
-			parser.plugin("program", function(ast) {
-				var firstNode = ast.body[0];
-				var dep;
+class UseStrictPlugin {
+	apply(compiler) {
+		compiler.plugin("compilation", (compilation, params) => {
+			params.normalModuleFactory.plugin("program", (ast) => {
+				let firstNode = ast.body[0];
```

Comment:
My opinion about that aligns with Mathias Bynens, who wrote a [nice article](https://mathiasbynens.be/notes/es6-const) about that.